SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialEventTracer.cpp

```diff
@@ -210,7 +210,7 @@ void SpatialEventTracer::UpdateComponent(const Worker_Op& Op)
 
 	Trace_SpanId MergeCauses[2] = { Op.span_id, OldSpanId };
 	TOptional<Trace_SpanId> SpanId = CreateSpan(MergeCauses, 2);
-	TraceEvent(FSpatialTraceEventBuilder::CreateMergeComponent(Id.EntityId, Id.ComponentId), SpanId);
+	TraceEvent(FSpatialTraceEventBuilder::CreateMergeComponentUpdate(Id.EntityId, Id.ComponentId), SpanId);
```

Comment:
Are these changes meant to be in this PR?

Reply:
Yes and no... The reason I changes this into two events was to make it easier to test.